test/mke2e/e2e_test.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mke2e
+
+import (
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/clientcmd"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	kueue "sigs.k8s.io/kueue/apis/kueue/v1beta1"
+	utiltesting "sigs.k8s.io/kueue/pkg/util/testing"
+	"sigs.k8s.io/kueue/test/util"
+)
+
+// +kubebuilder:docs-gen:collapse=Imports
+
+var _ = ginkgo.Describe("MultiKueue", func() {
+	var (
+		managerNs *corev1.Namespace
+		worker1Ns *corev1.Namespace
+		worker2Ns *corev1.Namespace
+	)
+	ginkgo.BeforeEach(func() {
+		managerNs = &corev1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				GenerateName: "multikueue-",
+			},
+		}
+		worker1Ns = managerNs.DeepCopy()
+		worker2Ns = managerNs.DeepCopy()
+		gomega.Expect(k8sManagerClient.Create(ctx, managerNs)).To(gomega.Succeed())
+		gomega.Expect(k8sWorker1Client.Create(ctx, worker1Ns)).To(gomega.Succeed())
+		gomega.Expect(k8sWorker2Client.Create(ctx, worker2Ns)).To(gomega.Succeed())
+
+	})
+
+	ginkgo.AfterEach(func() {
+		gomega.Expect(util.DeleteNamespace(ctx, k8sManagerClient, managerNs)).To(gomega.Succeed())
+		gomega.Expect(util.DeleteNamespace(ctx, k8sWorker1Client, worker1Ns)).To(gomega.Succeed())
+		gomega.Expect(util.DeleteNamespace(ctx, k8sWorker2Client, worker2Ns)).To(gomega.Succeed())
+	})
+	ginkgo.When("Using multiple clusters", func() {
+		ginkgo.DescribeTable("Create workload in cluster", func(c *client.Client, ns **corev1.Namespace) {
+			wl := utiltesting.MakeWorkload("wl", (*ns).Name).Obj()
+			gomega.Expect((*c).Create(ctx, wl)).To(gomega.Succeed())
+		},
+			ginkgo.Entry("manager", &k8sManagerClient, &managerNs),
+			ginkgo.Entry("worker1", &k8sWorker1Client, &worker1Ns),
+			ginkgo.Entry("worker2", &k8sWorker2Client, &worker2Ns),
+		)
+
+		ginkgo.DescribeTable("Cluster kubeconfig propagation", func(c *client.Client, key string, ns **corev1.Namespace) {
```

Comment:
Same question, what is this testing?

It looks like it's creating a client based on the secret. But this is not going through the manager cluster in any way. So we are just testing that we can create clients?

Reply:
I'll remove them in #1313, even without them we'l, need some other place holders.

Reply:
Isn't "Create workload in cluster" enough as a placeholder?

Reply:
the second table has a closer behavior to what we'll need in the future as is also uses the created secret.

Reply:
I still don't see the point, but we can proceed, given that you have a WIP for the next PR.